docs/src/apimanual.md

```diff
@@ -1012,38 +1012,53 @@ By convention, optimizers should not be exported and should be named
 
 ### Testing guideline
 
-The skeleton below can be used for the wrapper test file of a solver name `FooBar`. A few bridges are used to give examples, you can find more bridges in the [Bridges](@ref) section.
+The skeleton below can be used for the wrapper test file of a solver named `FooBar`.
 ```julia
 using MathOptInterface
 const MOI = MathOptInterface
 const MOIT = MOI.Test
+const MOIU = MOI.Utilities
 const MOIB = MOI.Bridges
 
-const optimizer = FooBarOptimizer()
+import FooBar
+const optimizer = FooBar.Optimizer()
+MOI.set(optimizer, MOI.Silent(), true)
+
+@testset "SolverName" begin
+    @test MOI.get(optimizer, MOI.SolverName()) == "FooBar"
+end
+
+@testset "supports_default_copy_to" begin
+    @test MOIU.supports_default_copy_to(optimizer, false)
+    # Use `@test !...` if names are not supported.
+    @test MOIU.supports_default_copy_to(optimizer, true)
+end
+
+const bridged = MOIB.full_bridge_optimizer(optimizer, Float64)
```

Comment:
Explain why `full_bridge_optimizer` is often needed to pass tests, or in what situations a solver could pass tests without a bridge.